README.md

```diff
@@ -340,6 +340,43 @@ store.dispatch('dec')        // Unknown event
 In order to work properly for imports, it is considering adding
 `allowSyntheticDefaultImports: true` to `tsconfig.json`.
 
+## Server-side rendering
+
+In order to preload data for server-side rendering, Storeon provide `customContext` function to create your own `useStoreon` hooks that it will depends on your custom context.
+
+```tsx
+// parent.tsx
+import { createContext, render } from 'react' // or preact
+
+import { createStoreon, StoreonModule } from 'storeon'
+import { customContext } from 'storeon/react' // or storeon/preact
+
+const CustomContext = createContext(store)
+
+// useStoreon will automatically recognize your storeon store and event types
+export const useStoreon = customContext(CustomContext)
+
+render(
+  <CustomContext.Provider value={store}>
+    <Counter />
+  </CustomContext.Provider>,
+  document.body
+)
+```
+
+```tsx
+// children.tsx
```

Comment:
I think we do not need TS types for example here. Let’s focus on SSR only since it is pretty expected behavior and we described it in the TS section.

Reply:
This should do it. 5ac3f21c83e1bc16b0336a372c9d4707efa8100a